protos/protos/public/uac/Collaborator.proto

```diff
@@ -69,6 +69,15 @@ message GetCollaborator {
     }
 }
 
+message RemoveAllCollaborators {
```

Comment:
Can we:
1. Move to the role service, since it's possible there are other associations that are not collaborators
2. Change the endpoint to "RemoveResource"? For example, it's possible a role is attached to the resource. This API would remove all references to the resource, which include collaborators but also other potential connections.